app/dashboard/utils.py

```diff
@@ -249,10 +249,10 @@ def get_bounty(bounty_enum, network):
 
 
 # processes a bounty returned by get_bounty
-def web3_process_bounty(bounty_data):
+def web3_process_bounty(bounty_data, dry_run=False):
     did_change, old_bounty, new_bounty = process_bounty_details(bounty_data)
 
-    if did_change and new_bounty:
+    if did_change and new_bounty and not dry_run:
```

Comment:
the problem here is that `process_bounty_changes()` does a lot more than just send the email..

Reply:
correct, the idea is to use `sync_geth` command to call `process_bounty_changes()` only.

to make code clear, i could update `sync_web3()` using `process_bounty_details()` instead of `web3_process_bounty()`